spec/type_aliases/file_source_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
```

Comment:
we should also remove spec/type_aliases/filesource_spec.rb and spec/type_aliases/filemode_spec.rb

Reply:
Since they are still aliases, I kept them around. Kind of not sure what to do with it though.